DDD.Sessionize.Tests/TestHelpers/TableStorageRepositoryMock.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using DDD.Core.AzureStorage;
+using Microsoft.WindowsAzure.Storage.Table;
+
+namespace DDD.Sessionize.Tests.TestHelpers
+{
+    public class TableStorageRepositoryMock<T> : ITableStorageRepository<T> where T : class, ITableEntity, new()
```

Comment:
Nice. Hand rolled test double is way more powerful and cleaner than with using a mocking library.